import-export-cli/cmd/importAPI.go

```diff
@@ -192,6 +198,66 @@ func mergeAPI(apiDirectory string, environmentParams *params.Environment) error
 	return nil
 }
 
+// Set security parameters 
+// @param environmentParamsEndpoints : Environment endpoint parameters from api_params.yaml
+// @param api : Parameters from api.yaml
+// @return error 
+func setSecurityEndpointsParams(environmentParamsEndpoints *params.EndpointData, api *gabs.Container) error {
+	// If the user has set (either true or false) the endpointSecured field in api_params.yaml the 
+	// following code should be executed. (if not set, the security endpoint settings will be made 
+	// according to the api.yaml file as usually)
+	// In Go, irrespective of whether a boolean value is "" or false, it will contain false by default.
+	// That is why, here a string comparison was  made since strings can have both "" and "false"
+	if environmentParamsEndpoints.EndpointSecured != "" {
+		// Convert the string endpointSecured to boolean
+		boolEndpointSecured, err := strconv.ParseBool(environmentParamsEndpoints.EndpointSecured)
+		if err != nil {
+			return err
+		}
+		if _, err := api.SetP(boolEndpointSecured, "endpointSecured"); err != nil {
+			return err
+		}
+		if boolEndpointSecured {
+			// If endpoint security is enabled, check whether the username and password has set in api_params.yaml
```

Comment:
Better to put the logic inside this if condition into a private function to improve readability. This reduces the level of nesting which makes the logic clear and less error prone